packages/react-core/src/next/components/Dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import styles from '@patternfly/react-styles/css/components/Dropdown/dropdown';
+import { css } from '@patternfly/react-styles';
+import { Menu, MenuContent, MenuProps } from '../../../components/Menu';
+import { Popper } from '../../../helpers/Popper/Popper';
+
+export interface DropdownProps extends MenuProps {
+  /** Anything which can be rendered in a dropdown */
+  children?: React.ReactNode;
+  /** Classes applied to root element of dropdown */
+  className?: string;
+  /** Renderer for a custom dropdown toggle. Forwards a ref to the toggle. */
+  toggleRender?: (toggleRef: React.RefObject<any>) => React.ReactNode;
+  /** Flag to indicate if menu is opened.  If is open */
+  isOpen?: boolean;
+  /** Function callback called when user selects item */
+  onSelect?: (event?: React.MouseEvent<Element, MouseEvent>, itemId?: string | number) => void;
+  /** Callback to for when the component needs to change the open state of the menu.
+   * If this is not provided the component will not close the menu when tab or escape are clicked. */
+  onIsOpenChange?: (isOpen: boolean) => void;
```

Comment:
Remove 'to' from description.
The prop name/description is a little unclear to me. If this is representing the situations when the menu is closing (via keypress or click out) like it appears, would `onClose` or `shouldClose` work better? 

Or have some extra props and create `onTab(isOpen)`, `onEscape(isOpen)`, and `onOutsideClick(isOpen)` to break out each separate behavior for the user in case they want finer control? We can also potentially leave the closing behavior as the default and only override it when the props are present rather than the other way around.

Reply:
Yeah this is a tough one.  I named it that because it allows the change the open status as related to the `isOpen` prop.    On close does not describe that.  Neither do the  `onTab(isOpen)` or  `onEscape(isOpen)`function names.

Reply:
I updated the comment.  I don't know if that is more clear.